Specs/DataSources/ModelGraphicsSpec.js

```diff
@@ -1,5 +1,6 @@
 defineSuite([
```

Comment:
There should also be tests for `Cesium3DTileset` (b3dm and i3dm) and `Model`. It should be enough to check that the pixel color changed after setting the lighting values.